test/test_linalg.py

```diff
@@ -1019,6 +1019,58 @@ def test_nuclear_norm_exceptions_old(self, device):
         self.assertRaisesRegex(RuntimeError, "duplicate or invalid", torch.norm, x, "nuc", (0, 0))
         self.assertRaisesRegex(IndexError, "Dimension out of range", torch.norm, x, "nuc", (0, 2))
 
+    @skipCUDAIfNoMagma
+    @skipCPUIfNoLapack
+    @dtypes(torch.float, torch.double, torch.cfloat, torch.cdouble)
+    def test_svd_compute_uv(self, device, dtype):
```

Comment:
This test and test_svd_no_compute_uv need to test more sizes, include a batched input.

Reply:
does it? The actual logic of svd is inside `_svd_helper`, which is already tested explicitly by `_test_svd_helper` and all the tests which calls it. In my idea, the `test_linalg_svd_*` are meant to test the only actual differences, i.e. the signature and return types.
But I am fine to duplicate all the tests if you think it's a good idea.